### ISSUE TITLE
Document all the Record_* classes in twisted.names.dns

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2936]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2936|
|Type|enhancement|
|Created|2007-12-05 14:11:22Z|
|Branch|https://github.com/twisted/twisted/tree/record-docs-2936|

Most of them are completely undocumented, including their attributes, many of which are quite opaque (for example, Record_SOA.mname).

<details><summary>Searchable metadata</summary>

```
trac-id__2936 2936
type__enhancement enhancement
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__branches_record_docs_2936 branches/record-docs-2936
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__ 
time__1196863882000000 1196863882000000
changetime__1197303120000000 1197303120000000
version__None None
owner__ 
cc__therve
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [22047]) Branching to 'record-docs-2936'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

ready for review

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

```Record_SRV``` and ```Record_MINFO``` have lost their EXPERIMENTAL comment.

Everything else is great.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [22052]) experimental

refs [#2936](https://github.com/twisted/twisted/issues/2936)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun removed owner|
|-|-|

Fixed that.  Not sure if SRV is really still considered experimental, but I won't try to change that for this ticket.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve set owner to @exarkun|
|-|-|

Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [22055]) Merge record-docs-2936

Author: exarkun
Reviewer: therve
Fixes [#2936](https://github.com/twisted/twisted/issues/2936)

Add documentation for all of the undocumented record types in
twisted.names.dns.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `reopened`|
|-|-|

(In [22057]) Revert r22055 - API documentation generation failure

reopens [#2936](https://github.com/twisted/twisted/issues/2936)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

I fixed the epytext markup.  **@see** was done wrong.  A build on the doc builder just passed with the current version.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_10"></a>|@therve set owner to @exarkun|
|-|-|

Hum sorry about that, a misconfiguration of pydoctor. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set status to `closed`|
|-|-|

(In [22062]) Merge record-docs-2936

Author: exarkun
Reviewer: therve
Fixes [#2936](https://github.com/twisted/twisted/issues/2936)

Add documentation for all of the undocumented record types in
twisted.names.dns.  This re-merge uses the @see epytext markup
correctly.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
